### PR TITLE
API to display policy xml content

### PR DESCRIPTION
**Pull request by shlomizadok**
Not sure if needs to merge to this branch.
Creates api to policy API.

**Should be merged also to master branch**
(may need some changes in routes.rb and  policies_controller.rb )

### CODE REVIEW

File: app/models/concerns/foreman_openscap/host_extensions.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def policies_enc
     end
 
     def combined_policies
-      combined = self.policies + self.hostgroup.policies
+      combined = self.hostgroup ? self.policies + self.hostgroup.policies : self.policies
```
Comment (shlomizadok) [inline app/models/concerns/foreman_openscap/host_extensions.rb]: This should be ported also to master branch

File: app/models/concerns/foreman_openscap/policy_extensions.rb [hunk 2]
```diff
@@ -134,6 +134,7 @@ def to_enc
         'id' => self.id,
         'profile_id' => self.scap_content_profile.try(:profile_id) || '',
         'content_path' => "/var/lib/openscap/content/#{self.scap_content.digest}.xml",
+        'download_path' => "/compliance/policies/#{self.id}/content" # default to proxy path
```
Comment (shlomizadok) [inline app/models/concerns/foreman_openscap/policy_extensions.rb]: Added in https://github.com/OpenSCAP/puppet-foreman_scap_client/pull/11

**Comment by shlomizadok** [comment]
@ares, @ohadlevy - Would love to have your take on this

**Comment by isimluk** [comment]
This should first go to master branch. And only after that being backported to maintenance branch. This process called backward patching.

I agree that opposite approach (forward patching) has it merits. The problem is that I have been forced to start with backward paching for historical reasons.

**Comment by shlomizadok** [comment]
#99 adds this functionality to master branch

**Comment by isimluk** [comment]
This can be merged.

However, I would wait until it is clear that we need this in the maintenance branch.

Thanks!

**Pull request merged**